### ISSUE TITLE
Add Dark Ritual

### ISSUE THREAD

**Issue by aok17** [jonathanpaulson/minions#28]
Raise zombie x3, except it's a sorcery.

**Comment by lightvector** [comment]
Can the zombies chain off of each other, with each one next to the previous?

**Comment by aok17** [comment]
No, they all have to be off the same unit. Also chaining raise zombie would be inconsistent with the new 'raise zombie' (you can no longer raise zombie off of a 'done acting' unit)
